config/jobs/kubernetes-sigs/controller-tools/OWNERS

```diff
@@ -1,7 +1,12 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/devel/owners.md
+# Note: This is kept in sync with: https://github.com/kubernetes-sigs/controller-tools/blob/master/OWNERS_ALIASES
```

Comment:
Can you add this to the commit message description? Or even better, explain in the commit message why we need to keep this in sync with https://github.com/kubernetes-sigs/controller-tools/blob/master/OWNERS_ALIASES? Is there any other context/history that could be useful here for future readers?

Reply:
Added a note in both places. Basically the idea is that approvers/reviewers of the respective repositories have the same rights for the jobs corresponding to the repositories.